pkg/event/manager.go

```diff
@@ -230,6 +235,82 @@ func postHookEvent(event *api.Event) {
 	log.Info("posthook of event")
 	log.Infof("worker： %s", event.Project.Worker)
 	terminateEventWorker(event)
+
+	pipeline := event.Pipeline
+	record := event.PipelineRecord
+
+	// post hooks
+	if pipeline != nil && pipeline.Notification != nil {
+
+		policy := pipeline.Notification.Policy
+
+		sendFlag := false
+		switch record.Status {
+		case api.Success:
+			if policy == "always" || policy == "success" {
```

Comment:
Hard code here is not very good, it would be better to define a `NotificationPolicyType` type.